qmlti/src/resources/LTI_Data_Connector_qmp.php

```diff
@@ -880,7 +940,7 @@ public function User_loadUsers($consumer_key) {
 
 /*
  * Load the user from the database
```

Comment:
Missing PHPDoc. Probably a good idea if someone else has to maintain this in the future.